jenkins/tenant/aws/ecoeng/PolicyJenkinsfileDaily

```diff
@@ -0,0 +1,80 @@
+account = []
```

Comment:
ecoeng_01
readme.md - group of coreos
ecoeng_02
readme.md - group of appeng

Reply:
ecoeng_02 or mikhail_02 ?

Reply:
named it as ecoeng02

Reply:
So please update it